### PR TITLE
docs: Install requirements for testing too

### PR DESCRIPTION
**Pull request by sils1297**

### CODE REVIEW

File: docs/Getting_Involved/Testing.rst [hunk 1]
```diff
@@ -13,7 +13,9 @@ To run tests, you first need to install some dependencies. This can be
 done by executing:
 
 ::
-
```
Comment (abhsag24) [inline docs/Getting_Involved/Testing.rst]: unnecessary?

File: docs/Getting_Involved/Testing.rst [hunk 2]
```diff
@@ -13,7 +13,9 @@ To run tests, you first need to install some dependencies. This can be
...
+    $ # Use coala-bears.git instead coala.git for testing bears
```
Comment (abhsag24) [inline docs/Getting_Involved/Testing.rst]: why not just give the full link, that's way better for a newcomer

**Comment by abhsag24** [comment]
ack 7feb69f, 5e9ee10

**Comment by abhsag24** [comment]
maybe @underyx  wants to see this before it is merged?

**Comment by sils1297** [comment]
waiting for @jschwarzwalder s response here, I guess she'll wake up soon

**Comment by jschwarzwalder** [comment]
It doesn't mention that you should be using Python3.

I don't know what coverage is, and what is normal and what should be reported.

**Comment by sils1297** [comment]
CC @jschwarzwalder for review

**Comment by jschwarzwalder** [comment]
Prerequisite - coala requires python 3 https://www.python.org/download/releases/3.0/. You will need to make sure that is installed

Check what version you have installed with `python -V`

To run tests, you first need to install some dependencies. This can be done by following these steps:

**Download the repository**
```
$ git clone https://github.com/coala-analyzer/coala.git
$ # Use https://github.com/coala-analyzer/coala-bears.git for coala bears
```
 
**move to the coala directory** 
```
$ cd coala  # cd coala-bears when testing bears
$ # Use pip for python 3 on windows

```
**pip these files that should install everything you need for the tests**
`$ pip3 install -r test-requirements.txt -r requirements.txt`

Let us know if you get any errors

**Comment by sils1297** [comment]
Ah, I like that, I feel like we might want to have a "Prerequisites" documentation page and link to that one as this will be needed in multiple places?

**Comment by jschwarzwalder** [comment]
I'll let you make the prereq page and link to it :grinning: 

I think I need to buy gitmate some chocolate first

**Comment by sils1297** [comment]
oooh this is so complicated :P I think the information should go into `Users/Tutorials/Development Setup` and we should add a link to it from several places. I don't have time for this right now though.

**Comment by sils1297** [comment]
what I wrote so far but it isn't really fitting to that page:

```rst
Setting up a Development Environment
====================================

In order to help develop coala, you will need the following prerequisites:

- Python 3.3 or higher installed on your PC (Check with ``python --version``)
- Clone the repository:
  ``git clone https://github.com/coala-analyzer/coala.git``, or
  ``git clone https://github.com/coala-analyzer/coala-bears.git`` if you want
  to develop the bears.
- Install the requirements:
  ``cd coala; pip install -r requirements.txt -r test-requirements.txt``, or cd
  into ``coala-bears`` respectively. Be sure that your pip really uses Python
  3, you might need to use ``pip3`` on some linux distributions.
```

**Comment by jschwarzwalder** [comment]
What do we want to do about this PR
Some options include:
1) add the prequisite information directly in the page for now 
2) leave off the prerequisite information and merge as is and address that later
3) make a new issue about the need for a prerequisite page,

**Comment by sils1297** [comment]
I'm for 3

**Comment by sils1297** [comment]
or somebody else can do 1
Sincerely,

Lasse Schuirmann

lasse@schuirmann.net
http://coala-analyzer.org/ - http://viperdev.io/ - http://gitmate.com/

**Comment by jschwarzwalder** [comment]
I've been trying to add the prerequisite information into the installation page, and as soon as that if finalized I think this can be merged and the other attempt I had made can be closed and deleted.

**Comment by sils1297** [comment]
Hey @Adrianzatreanu can you look into this and see what's needed so this can continue? AFAIK @jschwarzwalder is waiting for input from us here.

**Comment by sils1297** [comment]
Ping @Adrianzatreanu

**Comment by Adrianzatreanu** [comment]
looking now.

**Comment by Adrianzatreanu** [comment]
well as long as this is not moved to coala-docs it is really hard to tell what needs to be changed and what not, so please redo these commits in the other repo and ill take it from there :) or ping on gitter.

**Pull request closed without merging**